File: engine/model/src/main/scala/tools/packet/transfer/world/WorldTransferError.scala
package tools.packet.transfer.world

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class WorldTransferError(private var _error: Int) extends PacketInput {
  def error: Int = _error

  override def opcode(): SendOpcode = SendOpcode.CASHSHOP_CHECK_TRANSFER_WORLD_POSSIBLE_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/transfer/world/WorldTransferError.java
package tools.packet.transfer.world;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record WorldTransferError(Integer error) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_CHECK_TRANSFER_WORLD_POSSIBLE_RESULT;
   }
}